src/Aggregation/AdjacencyMatrix.php

```diff
@@ -26,25 +23,4 @@ public function setSeparator(string $separator = '&'): self
     {
         return $this->setParam('separator', $separator);
     }
-
-    public function toArray(): array
-    {
-        $array = [];
-        $filters = $this->getParam('filters');
-
-        foreach ($filters as $filter) {
-            $key = \key($filter);
-            $array['adjacency_matrix']['filters'][$key] = \current($filter)->toArray();
-        }
-
-        if ($this->hasParam('separator')) {
```

Comment:
Why can we get rid of of the separator and aggs stuff too?

Reply:
Yes as it will be handled by the inherited `toArray` method.
There are some tests that cover this in `AdjacencyMatrixTest`, behavior is same than before